### DISCUSSION TITLE
problems with start project..

### DISCUSSION THREAD

**Discussion by Damianeex** [nliaudat/floor-heating-controller#15] [Q&A] [answered]
Hi, I'm interested in your project but I'm having trouble using the YAML files. So far, I've been using one microcontroller (8266 / ESP32) for one .yaml file. In your repository, there are references to other YAML files, for example, from the config.yaml file to heating_channel_tanh.yaml. How should these files be configured or where should they be added so that the ESP32 works correctly? I'm using ESPHome integrated with Home Assistant.

**Comment by nliaudat** [comment, marked as answer]
You'll find all you need in https://esphome.io/guides/faq.html or https://esphome.io/components/packages.html
